src/test/java/org/example/dao/MySqlJobRoleDaoTest.java

```diff
@@ -43,6 +50,12 @@ public void tearDown() throws Exception {
     @Test
     public void getJobRoles_shouldReturnListOfJobRoles_whenDatabaseReturnsRowsOfJobRoles() throws JobRoleDaoException,
             Exception {
+
+        try (Statement stmt = h2Connection.createStatement()) {
+            stmt.execute("DROP TABLE IF EXISTS roles");
+        }
+
+        RunScript.execute(h2Connection, new FileReader("src/test/resources/schema.sql"));
```

Comment:
test should not set up schema - if you remove the DROP stuff into before each, this is not needed

Reply:
made this change